akka-docs/rst/common/may-change.rst

```diff
@@ -0,0 +1,49 @@
+
+.. _may-change:
+
+###########################
+Modules marked "May Change"
+###########################
+
+To be able to introduce new modules and APIs without freezing them the moment they
+are released because of our :ref:`BinCompatRules <binary compatibility guarantees>` we have introduced
```

Comment:
um, I thought "binary compatibility guarantees" is the text in the link, did I get the link syntax wrong?